### DISCUSSION TITLE
Telegram釣手手法，千萬不要傻傻截圖!

### DISCUSSION THREAD

**Discussion by ziihui** [1101-GS4719-C/Python-Community-GS4719-C#1174] [General]
前幾天就真的遇到ㄌ，重點我完全沒有起疑心，還認真跟他聊了起來xD
但我邊聊就很好奇為什麼我朋友要用兩個帳號跟我聊天，甚至就自己主動傳聊天室截圖給他~
幸運的是我的頁面沒有出現登入碼的資訊，不然帳號就要被盜惹好好笑
大家要注意壓><
新聞連結:https://udn.com/news/story/7088/6960671

**Comment by Lipunpun** [comment]
現在的年輕人真容易被騙，哪像我根本不會有人找我，邊緣到詐騙集團不知道我的電話號碼，厲害ㄅ。

**Comment by Jamesmoneya** [comment]
但最近的詐騙事件提醒了使用者需要保持警惕。我只是點開一個連結被莫名加了好幾個好友，嚇得我趕快全部刪掉

**Comment by Jack-1D** [comment]
其實我覺得這個詐騙手法蠻有趣的。1~2年前也有一種盜取帳號的手法，會利用好友傳送一個連結，並補上一句"這個是你嗎?"。如果點擊連結，帳號就會被盜。

**Comment by zilinng** [comment]
我覺得最好避免自己被盜的方法就是不明連結不要亂點！但就算知道道理我還是每次看到連結就手癢加上好奇併發,不點就全身不對勁….只能祈禱自己運氣好不要哪天真的被騙了…

**Reply by DuxtinTai** [reply]
說到詐騙的問題，我想問問各位，在通訊軟體會不會常常有一些來路不明的人來加好友或是追蹤，一般情況你們收到他們的訊息會回覆嗎?
先說我自己的情況是，會，但是我常常會有過高的防備心，很快就聊不下去了。

**Comment by Elaine2899** [comment]
前兩天也有一個親戚突然用Telegram密我，問我用Telegram多久了?知道這個程式可以拿來看電影嗎?然後就叫我截圖聊天室給她看。那時候剛好在準備考試沒讀，結果她就直接打電話給我，後來我就發現收到login code才發現她怪怪的，還好我平常都習慣塑膠。
